bundles/com.zeligsoft.domain.ngc.ccm.generator/src/com/zeligsoft/domain/ngc/ccm/generator/writers/AxciomaWriterSwitch.java

```diff
@@ -363,10 +363,15 @@ public Object caseStructType(StructType object) {
 			
 			// if AXCIOMA
 			String extensibility = "@final";
-			if(object.isIsAppendable()) {
+			if (object.isIsAppendable()) {
 				extensibility = "@appendable";
 			}
 			buf.append(String.format("%s%s%n", indentString, extensibility));
+			String nestedAnnotation = "@nested(FALSE)";
+			if (!nested) {
```

Comment:
This logic is confusing to me. While it might get the same result, I would rather the lines above, 346 & 347 reflect this inverse logic. i.e "If topLevel.matches("true") then nested = false" and visa-versa. Then the logic here can be simpler "If(nested) then annotation = "@nested(TRUE)""

Reply:
I've updated the logic, and added it to the AtcdWriter as well. I don't know if you are using that, but I figured it's probably best to keep those two generators in sync. Let me know if that's not ok. I'll cherry pick the change for v3.

Reply:
Yes, this logic looks good to me. We are not using ATCD anymore, but until we (or your team) decide to pull support for ATCD, you are right, we should keep them in sync

Reply:
ATCD has no support for `@nested`, it only has support for `@top-level` so you couldn't keep the version for AXCIOMA and ATCD in sync with respect to this annotation

Reply:
Oh. Ok, I'll retract the changes for ATCD.